src/blocks/weather.rs

```diff
@@ -290,6 +309,18 @@ impl WeatherResult {
 
             });
         }
+
+        if let Some(sunset) = self.sunset {
+            values.extend(map! {
```

Comment:
There's a new way to extend the map without creating a new temporary HashMap.

```rs
map! { @extend values
    "sunset" => Value::datetime(sunset, None)
}
```

or since you are just adding a single value you can use insert:

```rs
values.insert("sunset".into(), Value::datetime(sunset, None));
```

As things are right now both of these extends are missing a `;` which is causing the pre-commit checks to fail.